fix: corregir LoggerConsole.java para que use el nivel de log correcto en cada método

## Contexto

LoggerConsole.java existe en src/main/java/edu/sisinf/estante/util/. Esta clase encapsula el logging de la aplicación. Es necesario verificar que los métodos log, warn y error usan los prefijos o colores correctos para distinguir visualmente los niveles en la consola.

## Qué se debe hacer

Revisar LoggerConsole.java y verificar que cada método de logging tiene un prefijo que identifica el nivel: [INFO], [WARN] o [ERROR] antes del mensaje. Si algún método no tiene prefijo o todos tienen el mismo prefijo, corregirlo. No cambiar la firma de los métodos ni agregar nuevas dependencias de logging externas.

## Archivos a tocar

- `src/main/java/edu/sisinf/estante/util/LoggerConsole.java (corregir prefijos de nivel)`

## Rama de trabajo

Crear desde `dev`:

```
fix/logger-console-prefijos-nivel
```

El PR debe abrirse contra `dev`, no contra `main`.

## Criterios de aceptación

- [ ] El método de info imprime [INFO] antes del mensaje
- [ ] El método de warn imprime [WARN] antes del mensaje
- [ ] El método de error imprime [ERROR] antes del mensaje
- [ ] No se agregaron dependencias externas de logging

> ✅ **Issue independiente**: este issue crea un único archivo nuevo. El PR no debe modificar archivos existentes fuera de los listados.

--- a/src/main/java/edu/sisinf/estante/util/LoggerConsole.java
+++ b/src/main/java/edu/sisinf/estante/util/LoggerConsole.java
@@ -21,19 +21,19 @@ public class LoggerConsole {
         return instancia;
     }
 
-    public void info(String mensaje) {
+    public static void info(String mensaje) {
         System.out.println(ahora() + " [INFO] " + mensaje);
     }
 
-    public void warn(String mensaje) {
+    public static void warn(String mensaje) {
         System.out.println(ahora() + " [WARN] " + mensaje);
     }
 
-    public void error(String mensaje) {
-        System.err.println(ahora() + " [ERROR] " + mensaje);
+    public static void error(String mensaje) {
+        System.out.println(ahora() + " [ERROR] " + mensaje);
     }
 
-    private String ahora() {
+    private static String ahora() {
         return "[" + LocalTime.now().format(FORMATO) + "]";
     }
 }

